zanata-war/src/main/resources/db/changelogs/db.changelog-3.6.xml

```diff
@@ -30,4 +30,53 @@
       referencedTableName = "HProject" referencedColumnNames = "id"/>
   </changeSet>
 
+  <changeSet id="1" author="damason@redhat.com">
+    <comment>Create table HProject_LocaleAlias.</comment>
```

Comment:
Do we still want to keep the naming prefix with `H`? All the new tables has no prefix in it now.